app/src/main/java/org/mifos/mobilebanking/ui/activities/LoginActivity.java

```diff
@@ -125,7 +125,7 @@ public void showPasswordError(String error) {
     @OnClick(R.id.btn_login)
     public void onLoginClicked() {
 
-        final String username = tilUsername.getEditText().getEditableText().toString();
+        final String username = tilUsername.getEditText().getEditableText().toString().trim();
```

Comment:
@therajanmaurya Here, I have used string.trim() to remove leading and trailing spaces that can be added by user unintentionally.